.gitmodules

```diff
@@ -1,3 +1,6 @@
-[submodule "external/jsoncpp"]
-	path = external/jsoncpp
-	url = https://github.com/open-source-parsers/jsoncpp.git
+[submodule "external/rapidjson"]
+	path = external/rapidjson
+	url = https://github.com/Tencent/rapidjson.git
```

Comment:
you'll need to use my fork to get the fixes i did for mem hooking

Reply:
will need to look into that.  I *think* I'm using rapidjson in a way that it doesn't allocate anything but I will do some tests.

---
In reply to: [604527591](https://github.com/jplafonta/XPlatCSdk/pull/1#discussion_r604527591) [](ancestors = 604527591)

Reply:
I added a task to do a memory analysis similar to what you did for XSAPI.  Going to punt this for now.

---
In reply to: [604538948](https://github.com/jplafonta/XPlatCSdk/pull/1#discussion_r604538948) [](ancestors = 604538948,604527591)